frontend/package.json

```diff
@@ -33,6 +33,7 @@
     "@types/react": "18.2.20",
     "@types/react-dom": "18.2.7",
     "@vitejs/plugin-react": "4.0.4",
+    "esbuild": "0.19.2",
```

Comment:
```suggestion
``` 
You should uninstall it. To make it work without it, delete all node_modules, than install it again,